docs/reference/indices/dangling-index-import.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[dangling-index-import]]
+=== Import dangling index API
+++++
+<titleabbrev>Import dangling index</titleabbrev>
+++++
+
+Imports a dangling index.
+
+[[dangling-index-import-api-request]]
+==== {api-request-title}
+
+[source,console]
+--------------------------------------------------
+POST /_dangling/<index-uuid>?accept_data_loss=true
+--------------------------------------------------
+// TEST[skip:Difficult to set up]
+
+[[dangling-index-import-api-desc]]
+==== {api-description-title}
+
+include::{es-repo-dir}/indices/dangling-indices-list.asciidoc[tag=dangling-index-description]
+
+This API lets you import a single index into the cluster. You reference the
+index by its UUID, which you can find using the
+<<dangling-indices-list,list API>>. You can also
+<<dangling-index-delete,delete>> a dangling index.
+
+[[dangling-index-import-api-path-params]]
+==== {api-path-parms-title}
+
+`<index-uuid>`::
+(Required, string)
+UUID of the index to import. You can find this using the
+<<dangling-indices-list,list API>>.
+
+[[dangling-index-import-api-query-params]]
+==== {api-query-parms-title}
+
+`accept_data_loss`::
+(Required, boolean)
+This field must be set to `true` in order to carry out the import. {es}
+does not know where the dangling index data came from, nor does it know which shard copies are fresh and which are stale,
+so it can offer no guarantees about the data being
+imported.
```

Comment:
```suggestion
This field must be set to `true` to import a dangling index. Because {es}
cannot know where the dangling index data came from or determine which shard copies are fresh and which are stale,
it cannot guarantee the data being
imported.
```